5001-6000/LIT5844MagicPrayer.xml

```diff
@@ -31,19 +31,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p>Generic record for magic prayers against Zār, when no further information is available.</p>
```

Comment:
I think there is a silent "to be used" preceding "when no further…". Maybe better to write it out?